packages/ui-react/src/components/UserMenu/UserMenu.tsx

```diff
@@ -63,74 +65,77 @@ const UserMenu = ({
   }, [onClick, navigate, accountController]);
 
   return (
-    <ul onFocus={onFocus} onBlur={onBlur} className={styles.menuItems}>
-      {profilesEnabled && selectProfile && (
-        <ProfilesMenu
-          profiles={profiles ?? []}
-          currentProfile={currentProfile}
-          small={small}
-          selectingProfile={!!isSelectingProfile}
-          selectProfile={selectProfile}
-          createButtonLabel={t('nav.add_profile')}
-          switchProfilesLabel={t('nav.switch_profiles')}
-          onCreateButtonClick={() => navigate(PATH_USER_PROFILES_CREATE)}
-        />
-      )}
-      <li className={styles.sectionHeader}>{t('nav.settings')}</li>
-      {profilesEnabled && currentProfile && (
-        <li>
-          <MenuButton
+    <>
+      <h2 className={styles.sectionHeader} id={titleId}>
```

Comment:
We've internally discussed that we will not merge this change/PR yet, because currently `inert` is not correctly applied yet, causing the backdrop to still be accessible/scrollable and therefore the situation arises where this `<h2>` is in conflict with the overall HTML structure. If the `inert` of the sidebar is fixed (one of my tickets), this change can be applied.

Reply:
The h2 somehow doesn't feel "right", but it seems like the best approach here. I checked with Axe devtools and it didn't complain. The h2 is enclosed in a `<section>`. So it should be alright.

Reply:
It's a header of the section, and level 2 seems the only appropriate, so to me it seems the best option.